### PR TITLE
Endoc 454-update images and new procedures

### PR DESCRIPTION
**Pull request by jyunmitch**

### CODE REVIEW

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 1]
```diff
@@ -1,108 +1,89 @@
...
 
-    -   **Join Group.** Use the drop-down list to choose the
-        available user group to share the content with and select +. The group
-        selected will only be able to share and can not
-        administrate this content. This field is not mandatory.
+    *   `Categories`: Use `+` under `Join` to add to the applicable category tree for use in searching and sorting.  This field is not mandatory.
 
-    -   **Categories.** Select the category(s) and select + to add a classification to the 
-        content to use for searching and sorting.  This field is not mandatory.
+    *  `Content Attributes`: Define each required `Attributes` field and any optional fields for the `Content Type` of this item. Only the `Date` and `Title` are required.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:34]: this is awkward/confusing. maybe rephrase to: "It is mandatory to define the `Date` and `Title` of the `Content Type`. You may also define any optional `Attributes` fields."

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 2]
```diff
@@ -1,108 +1,89 @@
...
 
-    -   **Content Attributes.** Define each required attributes field and any optional fields
-        defined by the content types used for this Content. This field is mandatory.
+    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:36]: can omit "all"

would split into two sentences "This displays the saved iterations of the `Content` item. You can view details or revert to a previous version." -- a little more direct, which is good for bulleted parameter definitions.

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 3]
```diff
@@ -1,108 +1,89 @@
...
-        defined by the content types used for this Content. This field is mandatory.
+    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version. 
 
-![image](./img/Content3.png)
+![Add Content Information](./img/content-addinfo.png)
 
-Save the new content by selecting:
+## Saving and Approving Content 
+1. For new or edited content, determine which `Status` it belongs to:  
+`Status`:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:42]: i think you can omit this; the previous line implies the definitions apply to `Status`

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 4]
```diff
@@ -1,108 +1,89 @@
...
-    -   **Content Attributes.** Define each required attributes field and any optional fields
-        defined by the content types used for this Content. This field is mandatory.
+    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version. 
 
-![image](./img/Content3.png)
+![Add Content Information](./img/content-addinfo.png)
 
-Save the new content by selecting:
+## Saving and Approving Content 
+1. For new or edited content, determine which `Status` it belongs to:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:41]: would say "..determine the appropriate item `Status`:"

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 5]
```diff
@@ -1,108 +1,89 @@
...
+    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version. 
 
-![image](./img/Content3.png)
+![Add Content Information](./img/content-addinfo.png)
 
-Save the new content by selecting:
+## Saving and Approving Content 
+1. For new or edited content, determine which `Status` it belongs to:  
+`Status`:
+ * `Draft`: The content is in the development stage and not ready for approval or publication. The `Save` or `Save and Continue` buttons establish this status for all content.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:43]: for all content? aren't we just talking about a single `Content` item?

same as above: specific instances of content are singular and should be capitalized/backticked

would say "establish `Draft` status" -- otherwise it can read as those buttons establish `Save` or `Save and Continue` status

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 6]
```diff
@@ -1,108 +1,89 @@
...
 
--   **Save**: The content will be saved in a draft version.
+ * `Ready`: The content is ready for review and has not yet been approved nor published.  Items with this status can be saved and approved later or published directly.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:45]: nor -> or

published directly is awkward. do you mean published immediately? or published directly *to* something?

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 7]
```diff
@@ -1,108 +1,89 @@
...
 
--   **Save and Continue**: The content will be saved and the editable form will be displayed.
+2. Saving options:
 
--   **Save and Approve**: The content will be saved, approved and published.
+-   **Save**: The content will be saved in a draft version.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:49]: should capitalize/backtick? -> `Draft` ??

why switch from backticks to bold for these options?

change from future to present tense: *will* -> *is*

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 8]
```diff
@@ -1,108 +1,89 @@
...
 
-## Finding Content
+-   **Save and Continue**: The content will be saved as a draft version and the editable form will be displayed.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:51]: same as above

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 9]
```diff
@@ -1,108 +1,89 @@
...
 
-You can list specific Content by using the filters:
+-   **Save and Approve**: The content will be saved, approved, and ready for use in an application.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:53]: same as above

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 10]
```diff
@@ -1,108 +1,89 @@
...
--   **Pending Changes**: All created Content that was saved by only clicking on the
-    **Save** or **Save and Continue** button. The content is not
-    approved and therefore is not ready to be published.
+## Finding Content
 
--   **Ready for Approval**: All created Content that is in **Ready** status,
-    and saved by clicking on the **Save** or **Save and Continue** button.
-    The content is ready for review but has not yet been approved and
-    therefore has not been published.
+Using the `Advanced Filters` feature, you can search for Content by name, type, category, group and status. \
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:58]: odd to capitalize Content here when it's lowercase elsewhere; should either be general "content" or specific "`Content`"

"and" -> "or"

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 11]
```diff
@@ -1,108 +1,89 @@
...
+![Content Filters](./img/content-filters.png)
 
--   **Published**: All created Content that has been saved and approved
-    by clicking on the **Save and Approve** button. This content has been
-    published.
+In the resulting table, contents are listed with these parameters: `Name`, `Author`
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:61]: contents -> `Contents`; not talking about general content or you'd say "content is"

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 12]
```diff
@@ -1,108 +1,89 @@
...
+![Content Table List View](./img/content-table-view.png)
 
-In the table list, contents are listed with several parameters: Name,
-Created by, Last Edited, Type, Created Date, Owner Group, Join Group,
-Status, Restrictions, Code, Actions. Fields can be added or deleted from view by selecting or unselecting them
-in the Columns drop-down menu.
+## Content Actions
 
-![image](./img/Content6.png)
+For all items in the `Content` table, the `Actions` drop-down menu provides these executable options:
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:68]: these -> the following (that's what we usually use so is consistent; also a bit more specific/formal)

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 13]
```diff
@@ -1,108 +1,89 @@
...
 
-## Content Actions
+* `Copy/Paste`: replicate the content
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:70]: capitalize first word after colon

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 14]
```diff
@@ -1,108 +1,89 @@
...
+
+* `Draft Version`: edit the unpublished draft version
 
-For all Content in the list, you have access to a set of actions
-through the **Actions** kebab button, such as:
+*  `Published Version`: edit or revert to a previous published version of the content
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:74]: same as above

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 15]
```diff
@@ -1,108 +1,89 @@
...
 
--   **Edit**: update the data fields in the content definition
+*  `Edit`: update the data fields in the content definition
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:76]: same as above

`Content`?

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 16]
```diff
@@ -1,108 +1,89 @@
...
+![Content Actions](./img/content-actions.png)
 
--   **Unpublish**: hide the content
 
-![image](./img/Content7.png)
+## Publish, Unpublish, or Delete Content
 
-All Content in the list can be saved to your local file system by clicking the
-**Download** button and selecting either CSV or XLS format.
+To `Publish`, `Unpublish`, or `Delete` content, apply the buttons above the table for checked items. This will take you to a confirmation page where a summary page displays the usage of the item in your application.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:85]: would remove comma after `Unpublish`

"apply the buttons" sounds awkward -- click on?

change from future to present tense

**Review by Lyd1aCla1r3 — COMMENTED** [review]
general comment: change from future to present tense when describing actions/processes

File: vuepress/docs/next/tutorials/compose/content-tutorial.md (hunk 12 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/compose/content-tutorial.md:68]: I don't think we need to be that exacting, when meaning is not impaired or it's uncomplicated.

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 17]
```diff
@@ -1,108 +1,89 @@
...
-in the Columns drop-down menu.
+## Content Actions
 
-![image](./img/Content6.png)
+For all items in the `Content` table, the `Actions` drop-down menu provides these executable options:
 
-## Content Actions
+* `Copy/Paste`: replicate the content
+
+* `Draft Version`: edit the unpublished draft version
```
[the remark this answers is not in the document]
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/compose/content-tutorial.md:72]: Again, general use

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 18]
```diff
@@ -11,54 +11,52 @@ This tutorial describes how to create, edit and publish content in an applicatio
...
+
+ * `Ready`: The `Content` is ready for review, but has not been approved or published.  Items with this status can be saved and approved later or published immediately.
 
 2. Saving options:
 
--   **Save**: The content will be saved in a draft version.
+-   `Save`: The content is saved in a `Draft` version.
 
--   **Save and Continue**: The content will be saved as a draft version and the editable form will be displayed.
+-   `Save and Continue`: The content is saved as a draft version and the editable form is displayed.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:49]: "draft" is `Draft` right above

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 19]
```diff
@@ -11,54 +11,52 @@ This tutorial describes how to create, edit and publish content in an applicatio
...
 
--   **Save and Approve**: The content will be saved, approved, and ready for use in an application.
+-   `Save and Approve`: The content is saved, approved, and ready for use in an application.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:51]: i think you did not use comma before "and" earlier

File: vuepress/docs/next/tutorials/compose/content-tutorial.md [hunk 20]
```diff
@@ -11,54 +11,52 @@ This tutorial describes how to create, edit and publish content in an applicatio
...
 
 
-## Finding Content
+## Find Content
 
-Using the `Advanced Filters` feature, you can search for Content by name, type, category, group and status. \
+Using the `Advanced Filters` feature, you can search for Content by name, type, category, group or status.
```
Comment (Lyd1aCla1r3) [inline vuepress/docs/next/tutorials/compose/content-tutorial.md:56]: like above comment, no comma here before "or"

File: vuepress/docs/next/tutorials/compose/content-tutorial.md (hunk 18 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/compose/content-tutorial.md:49]: it is being used in the general sense of the word.

File: vuepress/docs/next/tutorials/compose/content-tutorial.md (hunk 19 above)
Comment (jyunmitch) [reply vuepress/docs/next/tutorials/compose/content-tutorial.md:51]: it's optional so ok either way.

**Review by Lyd1aCla1r3 — APPROVED** [review]
